components/vf-card/vf-card.scss

```diff
@@ -35,12 +35,14 @@
 
 .vf-card__content {
   grid-column: 1 / -1;
-  padding: 16px 24px;
+  padding: 1rem;
```

Comment:
Should we use the spacing tokens instead?

Reply:
Yes! I'll go back and do these where applicable.